src/xcode/ENA/ENA/Source/Scenes/Events/OnBehalfCheckinSubmission/TraceLocationSelection/__tests__/OnBehalfScanQRCodeCellModelTests.swift

```diff
@@ -0,0 +1,59 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import XCTest
+@testable import ENA
+
+class OnBehalfScanQRCodeCellModelTests: CWATestCase {
+
+	func testGIVEN_CellModel_THEN_InitialStateIsAsExpected() {
+		// GIVEN
+		let cellViewModel = OnBehalfScanQRCodeCellModel()
+
+		// THEN
+		XCTAssertEqual(
+			cellViewModel.text,
+			AppStrings.OnBehalfCheckinSubmission.TraceLocationSelection.scanButtonTitle
+		)
+		XCTAssertEqual(
+			cellViewModel.textColorPublisher.value,
+			UIColor.enaColor(for: .textPrimary1)
+		)
+		XCTAssertEqual(
+			cellViewModel.iconImagePublisher.value,
+			UIImage(named: "Icons_qrScan")
+		)
+		XCTAssertEqual(
+			cellViewModel.accessibilityTraitsPublisher.value,
+			[.button]
+		)
+	}
+
+	func testGIVEN_CellModel_WHEN_SetEnabled_THEN_TextColorAndButtonStateChange() {
+		// GIVEN
+		let cellViewModel = OnBehalfScanQRCodeCellModel()
+
+		// WHEN
+		cellViewModel.setEnabled(false)
```

Comment:
is `true` the default and is represented by `testGIVEN_CellModel_THEN_InitialStateIsAsExpected`?

Reply:
Yes, exactly 👍